packages/harmony/src/components/popup/Popup.tsx

```diff
@@ -377,7 +378,11 @@ export const Popup = forwardRef<HTMLDivElement, PopupProps>(function Popup(
     }
   })
 
-  const rootStyle = { zIndex, position: fixed ? ('fixed' as const) : undefined }
+  const rootStyle = {
+    zIndex,
+    position: fixed ? ('fixed' as const) : undefined,
```

Comment:
is the `as const` for type things? could maybe add CSSObject as the type of the object to help with this as well?

Reply:
Not entirely sure, I just added `width`. But I can try adding the CSSObject type!

Reply:
Nah it wasn't happy with that :(